### PR TITLE
Add/Drop column exist check.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@siddontang  PTAL

### CODE REVIEW

File: ddl/ddl.go [hunk 1]
```diff
@@ -502,7 +502,13 @@ func (d *ddl) AddColumn(ctx context.Context, ti table.Ident, spec *AlterSpecific
 		return errors.Trace(ErrNotExists)
 	}
 
-	var col *column.Col
+	// Check whether added column has existed.
+	colName := spec.Column.Name
+	col := column.FindCol(t.Cols(), colName)
+	if col != nil {
+		return errors.Errorf("Try to add a column with the same name of an already exists column - %s", colName)
```
Comment (siddontang) [inline ddl/ddl.go]: the error message seems too long.

**Comment by siddontang** [comment]
test covered?

**Comment by qiuyesuifeng** [comment]
Yep, has already been covered in ddl_test.go

File: ddl/ddl.go (hunk 1 above)
Comment (qiuyesuifeng) [inline ddl/ddl.go]: How about `Add an existed column - %s`?

**Comment by siddontang** [comment]
LGTM

**Pull request merged**